flow/raster_cache.cc

```diff
@@ -142,7 +145,7 @@ static std::unique_ptr<RasterCacheResult> Rasterize(
 
   SkCanvas* canvas = surface->getCanvas();
   canvas->clear(SK_ColorTRANSPARENT);
-  canvas->translate(-cache_rect.left(), -cache_rect.top());
+  canvas->translate(-dest_rect.left(), -dest_rect.top());
```

Comment:
We still have a rounding problem when using ICTMs, but it is the same rounding problem that we are already living with.

I describe the current rounding problem here: https://github.com/flutter/engine/pull/33981#issuecomment-1154312036